install.sh

```diff
@@ -386,7 +386,7 @@ os_is_deb ()
 
 is_python3 ()
 {
-    if [[ ${python} == 'python3' ]]; then
+    if [[ ${python} == python3* ]]; then
         return 0
     fi
     if eval "${python} -c 'import sys; sys.exit(0) if sys.version_info >= (3,0) else sys.exit(1)'"; then
```

Comment:
I will remove change of this line, it's not related to this issue. It's used only by `deps_install()` above, when `${install_os} == 'debian'` to `apt-get install` python3 stuff if `--python=python3` is specified. With this change it would install default python3 when you specify python3.anythingelse, which, most likely, is not what one wants.

Also, the name of function is confusing, IMO. Something like `is_python3_installed()` would be better.

Reply:
Well I mostly have that line so that you can pass absolute paths to the python binary.